CHANGELOG.md

```diff
@@ -1,24 +1,26 @@
-Gobrake Changelog
-=================
+# Gobrake Changelog
 
-### master
+## master
 
-* Added the `DisableRemoteConfig` option. This option configures the remote
```

Comment:
@ChimanJain this line is gone from the changelog. We need to restore it since it's related to a public API change.